drivers/bluetooth/hci/Kconfig

```diff
@@ -55,6 +55,15 @@ config BT_SPI_INIT_PRIORITY
 	depends on BT_SPI
 	default 75
 
+config BT_SPI_BOOT_TIMEOUT_SEC
+	int "Seconds to wait for SPI device to report ready"
+	depends on BT_SPI
+	default 30
+	help
+	  Maximum duration for the SPI device to report ready through the
+	  `EVT_BLUE_INITIALIZED` HCI event. Default is 30 seconds to support
+	  a bootloader image swap on the controller.
```

Comment:
```suggestion
	  Maximum duration for the Zephyr Project hci_spi sample to report ready
	  through the `EVT_BLUE_INITIALIZED` HCI vendor event. Default is 30
	  seconds to support a bootloader image swap on the Controller.
```

Reply:
If this is Zephyr Specific hci driver and hci_spi sample, then should be emulate `CONFIG_BT_WAIT_NOP`  feature instead in the hci_spi sample?

Reply:
Referring to other Zephyr applications in the description of a Kconfig symbol is confusing IMO, and also not strictly accurate because the option also applies to the ST HCI controller.

Reply:
Let us add defnite `depends on` then.

Reply:
```suggestion
	  Maximum duration for a SPI Controller to report ready through the
	  Zephyr Project defined `EVT_BLUE_INITIALIZED` HCI vendor event.
	  Default is 30 seconds to support a bootloader image swap on the Controller.
```

Reply:
It is confusing to have ST specific `hci_spi` sample and emulate `EVT_BLUE_INITIALIZED` in the sample. Either we accept that `EVT_BLUE_INITIALIZED` is Zephyr Project defined, or be specific that `hci_spi` sample is expecting other vendors to emulate a ST SPI Controller behavior.